lib/qesapdeployment.pm

```diff
@@ -412,6 +461,7 @@ sub qesap_prepare_env {
     unless ($args{only_configure}) {
         qesap_create_folder_tree();
         qesap_get_deployment_code();
+        qesap_get_roles_code();
```

Comment:
This line has a big impact: it means that the new repo is cloned at each test that is using qe-sap-deployment, even if roles are not used yet in all tests